ui/shared/components/panel/variants/Transcripts.jsx

```diff
@@ -78,6 +78,10 @@ ConsequenceDetails.propTypes = {
   ensemblLink: PropTypes.object,
 }
 
+export const isManeSelect = (transcript, transcriptsById) => (
+  !!transcript.maneSelect || transcriptsById[transcript.transcriptId]?.isManeSelect
```

Comment:
What data is stored on the `transcript` and what is stored in `transcriptsById`?

Reply:
The best way to understand this is to actually look into the test fixtures, as the fields are pretty different depending on what the data source is - the new VEP annotations have different transcript annotations than old VEP. `transcriptsById` adds in the refeseq transcript ID and whether or not a transcript is MANE.With thelatest round of VEP there is nothing we need from `transcriptsById`, it is all moved into the new pipeline